app/services/movie_service.rb

```diff
@@ -27,13 +37,13 @@ def service
   def service_params
     {
       url: base_uri,
-      params: {api_key: ENV["moviedb_key"]}
+      params: { api_key: ENV["moviedb_key"] }
     }
   end
 
   def build_movie_objects(response)
-    JSON.parse(response.body)["results"].map do |movie|
+    JSON.parse(response.body, symbolize_names: true)[:results].map do |movie|
       Movie.new(movie)
     end
   end
```

Comment:
A potential refactor here could be pulling this functionality out and placing it in the `MovieFacade` per today's facade lesson - the service could be responsible for just returning `json` and the facade could be responsible for building `Movie` objects with that `json`

Reply:
Agreed, I left it as is because this was a method you wrote and didn't want to break your work, I definitely felt the pain of not having a central location for parsing, having to call it multiple times and add symbolize_names everywhere. I'll add it to the to do list for a refactor branch